### PR TITLE
Added ColorPicker under ColorButts

### PR DESCRIPTION
**Pull request by aldrenalangcao**
Implementation for this https://github.com/nakulrathore/Box-Shadows/issues/14

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -130,26 +131,30 @@ class Bloc extends Component {
...
+              className={background === bg.color ? "selected" : ""}
+              style={colorButtsStyle}
+            >
+              {bg.color}
+            </li>
+          );
+        })}
+      </ul>
+      <div className="ColorPickerContainer">
+        <HuePicker color={background} onChange={(color)=>onChangeBackground(color.hex)} />
```
Comment (jonny22094) [inline src/App.js]:
```suggestion
        <HuePicker color={background} onChange={(color) => onChangeBackground(color.hex)} />
```
Missing space

File: src/App.js [hunk 2]
```diff
@@ -130,26 +131,30 @@ class Bloc extends Component {
...
+        change background :
+        {backgroundColors.map((bg, index) => {
+          const colorButtsStyle = {
+            background: bg.color,
+            color: Color(bg.color).isDark() ? "#FFF" : "#000"
+          };
+          return (
+            <li
+              key={index}
+              onClick={()=>onChangeBackground(bg.color)}
```
Comment (jonny22094) [inline src/App.js]:
```suggestion
              onClick={() => onChangeBackground(bg.color)}
```
Missing space

**Comment by nakulrathore** [comment]
@aldrenalangcao , 
-the background switcher design and code is changed with this-> #23 
-also the slider doesn't give better control over background change , a simple color picker would be great, check this out https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/color

please add new color picker following new design if you can, on other merge request

File: src/App.js (hunk 1 above)
Comment (nakulrathore) [reply src/App.js]: thanx @jonny, but i will have to close this for now 
https://github.com/nakulrathore/Box-Shadows/pull/21#issuecomment-537919962

File: src/App.js [hunk 3]
```diff
@@ -130,26 +131,30 @@ class Bloc extends Component {
 
 const ColorButts = ({ background, onChangeBackground }) => {
   return (
```
Comment (jonny22094) [inline src/App.js]: You dont have logic in this component. So you can use implicit return
```
const ColorButts = ({ background, onChangeBackground }) => (
    <div className="ColorButts">
      <ul>
        change background :
        {backgroundColors.map((bg, index) => {
          const colorButtsStyle = {
            background: bg.color,
            color: Color(bg.color).isDark() ? "#FFF" : "#000"
          };
          return (
            <li
              key={index}
              onClick={()=>onChangeBackground(bg.color)}
              className={background === bg.color ? "selected" : ""}
              style={colorButtsStyle}
            >
              {bg.color}
            </li>
          );
        })}
      </ul>
      <div className="ColorPickerContainer">
        <HuePicker color={background} onChange={(color)=>onChangeBackground(color.hex)} />
      </div>
    </div>
  );

```

Comment (nakulrathore) [reply src/App.js]: @jonny
already implemented on #23

**Pull request closed without merging**